src/pages/blog/suggestions/index.mdx

```diff
@@ -0,0 +1,55 @@
+import { ArticleLayout } from '@/components/ArticleLayout'
+
+export const meta = {
+  title:
+    'Keep Your Documentation Fresh Through Intelligent Content Suggestions',
+  description: 'Using AI Recommendations to Improve Documentation',
+  mobileImageUrl: '/static/images/blog/suggestions/thumbnail.png',
+  desktopImageUrl: '/static/images/blog/suggestions/desktop.png',
+  author: 'Josh Kim',
+  authorRole: 'Head of Ops and Sales',
```

Comment:
I think in the future, we should just try to use the department name. Let's try for Operations if that works for you

Reply:
Changes to Operations